app/uk/gov/hmrc/testuser/services/TestUserService.scala

```diff
@@ -53,15 +61,25 @@ class TestUserService @Inject()(val passwordService: PasswordService,
     testUserRepository.createUser(organisation.copy(password = hashedPassword)) map {
```

Comment:
It would be nice to have this as a for comprehension with the individual steps broken out as functions to make this whole section more readable now it's getting more complex.

Reply:
no problem